web-client/public/js/update-app.js

```diff
@@ -212,52 +232,65 @@ export const updateApp = grnState => {
         $(LOCK_SLIDERS_MENU_OPTION + " span").addClass("invisible");
         $(LOCK_SLIDERS_BUTTON).prop("checked", false);
         $(RESET_SLIDERS_BUTTON).prop("disabled", false);
+        $(LINK_DIST_SLIDER_ID).prop("disabled", false);
+        $(CHARGE_SLIDER_ID).prop("disabled", false);
         $(RESET_SLIDERS_MENU_OPTION).parent().removeClass("disabled");
         $("#link-distance").parent().removeClass("disabled");
         $("#charge").parent().removeClass("disabled");
     }
 
     const resetValues = () => {
-        grnState.chargeSlider.backup = grnState.chargeSlider.currentVal;
-        grnState.linkDistanceSlider.backup = grnState.linkDistanceSlider.currentVal;
-        grnState.chargeSlider.currentVal = grnState.chargeSlider.defaultVal;
-        grnState.linkDistanceSlider.currentVal = grnState.linkDistanceSlider.defaultVal;
-        $("#charge-menu").val(grnState.chargeSlider.defaultVal);
-        $("#link-distance-menu").val(grnState.linkDistanceSlider.defaultVal);
+        CHARGE_BACKUP_VALUE = CHARGE_CURRENT_VALUE;
+        LINK_DIST_BACKUP_VALUE = LINK_DIST_CURRENT_VALUE;
+        CHARGE_CURRENT_VALUE = CHARGE_DEFAULT_VALUE;
+        LINK_DIST_CURRENT_VALUE = LINK_DIST_DEFAULT_VALUE;
```

Comment:
So to verify, you are storing these values in a separate location, for later use? If that is so, use camel case rather than all caps.